websites/B/Bounce/dist/metadata.json

```diff
@@ -11,12 +11,14 @@
     "ga_IE": "Is stáisiún ceoil idirnáisiúnta buailte é Bounce Radio a chuireann ceol buailte ar fáil do gach duine. Déanta ag gamers do gamers."
   },
   "url": [
-    "wearebounce.net",
-    "www.wearebounce.net"
+    "itsbounce.net",
+    "itsbounce.com",
+    "itsbounce.uk",
+    "wearebounce.co.uk"
   ],
-  "version": "2.5.3",
-  "logo": "https://boun.cc/cdn/noshadow.png",
-  "thumbnail": "https://i.imgur.com/XmJTiZY.png",
+  "version": "3.0.0",
+  "logo": "https://i.imgur.com/JgK4qYg.png",
+  "thumbnail": "https://i.imgur.com/tlLbekj.png",
   "color": "#32a852",
```

Comment:
thats not really a promotion image or a screenshot of the website homepage... the colors are not even related to the logo...